apps/services/notifications/infra/notifications.ts

```diff
@@ -0,0 +1,21 @@
+import { service, ServiceBuilder } from '../../../../infra/src/dsl/dsl'
+
+export const serviceSetup = (): ServiceBuilder<'notifications'> =>
+  service('notifications')
+    .image('services-notifications')
+    .namespace('notifications')
+    .serviceAccount('notifications')
+    .command('node')
+    .args('--tls-min-v1.0', 'main.js')
+    .env({
+      MAIN_QUEUE_NAME: 'notifications',
+      DEAD_LETTER_QUEUE_NAME: 'notifications-failure',
+      AWS_REGION: 'eu-west-1',
+    })
+    .secrets({
+      SQS_ACCESS_KEY: '/k8s/notifications/SQS_ACCESS_KEY',
```

Comment:
SQS_ACCESS_KEY and SQS_SECRET_ACCESS_KEY is unnecessary.
We need to create a irsa to give the service SQS permissions in the infra repo.

Reply:
ok, so skip it .. makes sense